UnitTests/TestReloadWorks.cs

```diff
@@ -1,35 +1,40 @@
-﻿using netmockery;
+﻿using Microsoft.AspNetCore.Mvc.Testing;
+using netmockery;
 using Newtonsoft.Json;
 using System;
-using System.Collections.Generic;
 using System.Linq;
+using System.Net;
 using System.Net.Http;
 using System.Threading.Tasks;
 using Xunit;
 
 namespace UnitTests
 {
-    public class TestReloadWorks : WebTestBase, IDisposable
+    public class TestReloadWorks : IDisposable
     {
-        EndpointCollectionProvider ecp;
-        DirectoryCreator dc;
+        private readonly DirectoryCreator _dc;

```

Comment:
Ingen grunn til å endre kodestandard her, dette blir bare kosmetiske endringer (gjelder også for TestResponseHasCorrectEncoding).

Reply:
Endret
- https://github.com/codeape2/netmockery/pull/22/commits/33bc11c7610620dd4dac679db874f0e55b322726
- https://github.com/codeape2/netmockery/pull/22/commits/8936c33c41fa118b0c1cc587cb556cdf91090f0b